src/core/meshcop/border_agent.hpp

```diff
@@ -157,6 +177,18 @@ class BorderAgent : public InstanceLocator, private NonCopyable
 
         typedef otBorderAgentEphemeralKeyCallback CallbackHandler; ///< Callback function pointer.
 
+        /**
+         * Represents the state of the `EphemeralKeyManager`.
+         */
+        enum State : uint8_t
+        {
+            kStateDisabled  = OT_BORDER_AGENT_STATE_DISABLED,  ///< Ephemeral key feature is disabled.
+            kStateStopped   = OT_BORDER_AGENT_STATE_STOPPED,   ///< Enabled, but the key is not set and started.
+            kStateStarted   = OT_BORDER_AGENT_STATE_STARTED,   ///< Key is set and listening to accept connection
+            kStateConnected = OT_BORDER_AGENT_STATE_CONNECTED, ///< Session connected, not full commissioner
```

Comment:
```suggestion
            kStateStarted   = OT_BORDER_AGENT_STATE_STARTED,   ///< Key is set and listening to accept connection.
            kStateConnected = OT_BORDER_AGENT_STATE_CONNECTED, ///< Session connected, not full commissioner.
```

Reply:
Thanks. Change all in the new pushed commit.